synapse/models/inet.py

```diff
@@ -695,487 +695,791 @@ def _revModl201710111553(self):
-                            'doc': 'The prop=valu of the account property that was changed. Valu should be the '
-                                   'old / original value, while the new value should be updated on the '
-                                   'inet:web:acct form.'}),
-                    ('pv:prop', {'ptype': 'str', 'doc': 'Property which changed', 'ro': 1}),
-                    ('pv:strval', {'ptype': 'str', 'ro': 1,
-                                   'doc': 'The normed value of the property (specified by pv), if the property is '
-                                          'a string', }),
+                        'doc': 'The prop=valu of the account property that was changed. Valu should be '
+                            'the old / original value, while the new value should be updated on the '
+                            'inet:web:acct form.'}),
+                    ('pv:prop', {'ptype': 'str', 'ro': 1,
+                        'doc': 'The property that was changed.'}),
                     ('pv:intval', {'ptype': 'int', 'ro': 1,
-                                   'doc': 'The normed value of the property (specified by pv), if the property is '
-                                          'a integer.', }),
+                        'doc': 'The normed value of the property (specified by pv), if the property is an integer.'}),
+                    ('pv:strval', {'ptype': 'str', 'ro': 1,
+                        'doc': 'The normed value of the property (specified by pv), if the property is a string.'}),
                 ]),
 
                 ('inet:web:logon', {'ptype': 'inet:web:logon'}, [
-                    ('acct', {'ptype': 'inet:web:acct', 'doc': 'The account associated with the logon event.', }),
-                    ('acct:site', {'ptype': 'inet:fqdn', }),
-                    ('acct:user', {'ptype': 'inet:user', }),
-                    ('time', {'ptype': 'time', 'doc': 'The time the account logged into the service', }),
-                    ('ipv4', {'ptype': 'inet:ipv4', 'doc': 'The source IPv4 address of the logon.'}),
-                    ('ipv6', {'ptype': 'inet:ipv6', 'doc': 'The source IPv6 address of the logon.'}),
-                    ('logout', {'ptype': 'time', 'doc': 'The time the account logged out of the service.'})
+                    ('acct', {'ptype': 'inet:web:acct', 'ro': 1, 'req': 1,
+                        'doc': 'The web account associated with the logon event.'}),
+                    ('acct:site', {'ptype': 'inet:fqdn', 'ro': 1,
+                        'doc': 'The site or service associated with the account.'}),
+                    ('acct:user', {'ptype': 'inet:user', 'ro': 1,
+                        'doc': 'The unique identifier for the account.'}),
+                    ('time', {'ptype': 'time',
+                        'doc': 'The date and time the account logged into the service.'}),
+                    ('ipv4', {'ptype': 'inet:ipv4',\
```

Comment:
Trailing `\`

Reply:
yea, lets remove the trailing \  as it's not needed

Reply:
fixed